Annotation/constituent_golden_set.json

```diff
@@ -232,14 +227,26 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "how DC might collect the various crossover miniseries specials",
                 "embedded clauses": []
             }
         ]
     },
     "179": {
         "sentence": "Whether Vengeance will include Sam Loeb's #26, and how DC will collect Mark Verheiden's issues after Jeph Loeb departs, remains to be seen.",
         "embedded clauses": [
+            {
+                "predicate": [
+                    {
+                        "str": "seen",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "polar",
+                "clause": "Whether Vengeance will include Sam Loeb's #26",

```

Comment:
Possibly this is the subject of the verb "remains" and therefore not an embedded clause, but the verb is acting as a linking verb the passive "seen" so I'm not sure if this then acts as the embedded clause for "seen". It's a tricky case

Reply:
I think this is one of those that we take to the supervisor, because I dont have a problem with it